feature/system/tests/osinstall/osinstall_test.md

```diff
@@ -0,0 +1,42 @@
+# gNOI-4.1: Software Upgrade
+
+## Summary
+
+Validate new software can be copied and activated on single and dual supervisors.
+
+## Procedure
```

Comment:
Just want to double check, did the test implement all the procedure here? The missing items should have a TODO in this file.

Reply:
Yes - there is one item I added a TODO for due to partial coverage but this should implement the test procedure.